daml-assistant/daml-helper/src/DA/Daml/Helper/Start.hs

```diff
@@ -168,12 +168,13 @@ withJsonApi (SandboxPort sandboxPort) (JsonApiPort jsonApiPort) extraArgs a = do
             , "--http-port", show jsonApiPort
             , "--allow-insecure-tokens"
             ] ++ extraArgs
+    putStrLn ("Starting JSON API: " <> unwords args)
     withPlatformJar args "json-api-logback.xml" $ \ph -> do
         putStrLn "Waiting for JSON API to start: "
         -- The secret doesn’t matter here
         let token = JWT.encodeSigned (JWT.HMACSecret "secret") mempty mempty
                 { JWT.unregisteredClaims = JWT.ClaimsMap $
-                      Map.fromList [("https://daml.com/ledger-api", Object $ HashMap.fromList [("actAs", toJSON ["Alice" :: T.Text]), ("ledgerId", "MyLedger"), ("applicationId", "foobar")])]
+                      Map.fromList [("https://daml.com/ledger-api", Object $ HashMap.fromList [("actAs", toJSON ["Alice" :: T.Text]), ("ledgerId", "sandbox"), ("applicationId", "foobar")])]
```

Comment:
This is a bit fishy. We really don’t know the ledger id. However in SDK 2.0 the ledger id will become optional. I suggest adding a TODO here to update this and link it to some issue.

Reply:
I was thinking of adding a --ledgerid option in daml start, but if it becomes optional we can just drop it.

Reply:
Added a TODO and linked to https://github.com/digital-asset/daml/issues/12145